chatterbot/utils/queues.py

```diff
@@ -0,0 +1,30 @@
+class ResponseQueue(object):
```

Comment:
Is there any reason why you aren't using the Python [`queue`](https://docs.python.org/3/library/queue.html) here?